manual/src/components/URLLoader.ts

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+
+export class URLLoader {
+  public urls: string[] = [];
+
+  constructor(public path: string) {}
+
+  /**********************************
+   * Read the URLS from a file.
+   * 
+   * 
+   * URL's should be separated by newlines after each line.
+   * EXAMPLE: (URL's shown are fake)
+   * 
+   * urls.txt - - - - - - - - - - - - - - - - - - - - - - -
+   * http://photobase.com/238946
+   * http://photobase.com/3u9256
+   * - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   *********************************/
+  public read(): void {
+    this.urls = fs.readFileSync(this.path, {
+      encoding: 'utf-8'
+    })
+    .split('\n')
```

Comment:
If you split(/\R/), the \r will be removed automatically.
see https://www.regular-expressions.info/refcharacters.html and look for Line Break